ansible/roles/host-ocp4-installer/files/cluster-network-03-config.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: operator.openshift.io/v1
```

Comment:
I admit I have not looked at this in detail, but can this not be set in the install-config? Or is it just not ready at this point in the 4.6 cycle?

Reply:
@stencell Currently, this extra manifest is required for 4.6. But will go away eventually when OVN becomes the default.

Reply:
Thanks for clarifying @christianh814